README.md

```diff
@@ -1 +1,122 @@
-# ramen
+<!---
+
+<img alt="Ramen" src="Documentation/media/logo.svg" width="50%" height="50%">
+
+[![CNCF Status](https://img.shields.io/badge/cncf%20status-graduated-blue.svg)](https://www.cncf.io/projects)
+[![Build Status](https://jenkins.ramen.io/buildStatus/icon?job=ramendr/ramen/master)](https://jenkins.ramen.io/blue/organizations/jenkins/ramendr%2Framen/activity)
+[![GitHub release](https://img.shields.io/github/release/ramendr/ramen/all.svg)](https://github.com/ramendr/ramen/releases)
+[![Docker Pulls](https://img.shields.io/docker/pulls/ramen/ceph)](https://hub.docker.com/u/ramen)
+[![Go Report Card](https://goreportcard.com/badge/github.com/ramendr/ramen)](https://goreportcard.com/report/github.com/ramendr/ramen)
+[![CII Best Practices](https://bestpractices.coreinfrastructure.org/projects/1599/badge)](https://bestpractices.coreinfrastructure.org/projects/1599)
+[![FOSSA Status](https://app.fossa.io/api/projects/git%2Bgithub.com%2Framendr%2Framen.svg?type=shield)](https://app.fossa.io/projects/git%2Bgithub.com%2Framendr%2Framen?ref=badge_shield)
+[![Slack](https://slack.ramen.io/badge.svg)](https://slack.ramen.io)
+[![Twitter Follow](https://img.shields.io/twitter/follow/ramen_io.svg?style=social&label=Follow)](https://twitter.com/intent/follow?screen_name=ramen_io&user_id=788180534543339520)
+
+-->
+
+# What is Ramen?
+
+Ramen is an open source **disaster-recovery service** for stateful applications running on
+OpenShift Container Storage (OCS).  Ramen provides a simplified interface to automate the life-cylce of an 
```

Comment:
Should we say that is running on OCS?  Right now it is one-to-one, but once we adapt scribe then that will not be true.